pkg/reconciler/trustroot/controller.go

```diff
@@ -78,22 +76,8 @@ func NewController(
 			pkgreconciler.NamespaceFilterFunc(system.Namespace()),
 			pkgreconciler.NameFilterFunc(config.SigstoreKeysConfigName)),
 		Handler: controller.HandleAll(grCb),
-	}, FromContextOrDefaults(ctx)); err != nil {
+	}, tuf.FromContextOrDefaults(ctx)); err != nil {
 		logging.FromContext(ctx).Warnf("Failed configMapInformer AddEventHandlerWithResyncPeriod() %v", err)
 	}
 	return impl
 }
-
-func ToContext(ctx context.Context, duration time.Duration) context.Context {
```

Comment:
Due to an import cycle, I moved the the trust root resync period `ToContext` and `FromContext` functions to the `tuf` package. I don't think it's too much of a stretch for these functions to live in that package as they are managing the TUF root related values.